tests/test_bug_features.py

```diff
@@ -163,6 +165,19 @@ def test_comment_length(read):
     read("comment_length.json", comment_length, [566, 5291])
 
 
+def test_bug_extractor():
+    feature_extractors = [blocked_bugs_number(), bug_reporter(), bug_reporter()]
+    cleanup_functions = [
+        fileref(),
+        url(),
+        url(),
+    ]
+    try:
+        BugExtractor(feature_extractors, cleanup_functions)
+    except AssertionError:
+        print("exception handled")
+
```

Comment:
There should be three tests:
1) Two feature extractors and two cleanup functions, without duplicates;
2) Two duplicate feature extractors and two non-duplicate cleanup functions;
3) Two non-duplicate feature extractors and two duplicate cleanup functions.